components/organisms/Layout.js

```diff
@@ -31,6 +30,10 @@ export const Layout = ({
   const { t } = useTranslation("common");
   const language = locale === "en" ? "fr" : "en";
 
+  useEffect(() => {
+    console.log("lang: ", Cookies.get("lang"));
+  }, []);
+
```

Comment:
Would you be able to remove this? We don't really want random console logs around the site.